rebar.config

```diff
@@ -13,10 +13,10 @@
   {lager, "2.0.3", {git, "git://github.com/basho/lager.git", {tag, "2.0.3"}}},
   {poolboy, ".*", {git, "git://github.com/basho/poolboy.git", {tag, "0.8.1p3"}}},
   {basho_stats, ".*", {git, "git://github.com/basho/basho_stats.git", {tag, "1.0.3"}}},
-  {riak_sysmon, ".*", {git, "git://github.com/basho/riak_sysmon.git", {tag, "2.0.1"}}},
-  {riak_ensemble, ".*", {git, "git://github.com/basho/riak_ensemble", {tag, "2.1.0"}}},
+  {riak_sysmon, ".*", {git, "git://github.com/basho/riak_sysmon.git", {branch, "develop"}}},
```

Comment:
Yea - figured fixing the rebar.config as part of the forward merge, rather than breaking develop, was reasonable.